### PR TITLE
feat: Cache all the kernels. No Akmods Bootstrap

### PR DESCRIPTION
**Pull request by m2Giles**
## Thank you for contributing to the Universal Blue project!

Please [read the Contributor's Guide](https://universal-blue.org/contributing.html) before submitting a pull request.

### CODE REVIEW

**Comment by bsherman** [comment]
Seems odd that the args flip flop... first last then first

image

**Comment by m2Giles** [comment]
Something interesting.

Asus doesn't provide kernel headers:

image

File: .github/workflows/reusable-build.yml [hunk 1]
```diff
@@ -27,58 +26,114 @@ jobs:
...
+                ;;
+              "main")
+                linux=$(skopeo inspect docker://quay.io/fedora-ostree-desktops/base:${{ matrix.fedora_version }} | jq -r '.Labels["ostree.linux"]' )
+                ;;
+              "coreos-stable")
+                linux=$(skopeo inspect docker://quay.io/fedora/fedora-coreos:stable | jq -r '.Labels["ostree.linux"]' )
+                coreos_fedora_version=$(echo $linux | grep -oP 'fc\K[0-9]+')
+                if [[ "${{ matrix.fedora_version }}" != coreos_fedora_version ]]; then
+                   major_minor_patch=$(echo $linux | cut -d - -f 1)
+                   linux="${major_minor_patch}-200.fc$(($coreos_fedora_version - 1)).$(uname -m)"
```
Comment (m2Giles) [inline .github/workflows/reusable-build.yml:97]: Should this use math for calculating the fedora version? I think not.

**Review by bsherman — APPROVED** [review]
Looks great! but @EyeCantCU has final say.

**Review by EyeCantCU — APPROVED** [review]
This is a significant improvement! Looks great to me

**Pull request merged**